src/hooks/useEvm/useEvm.ts

```diff
@@ -25,17 +28,19 @@ export const useEvm = () => {
   useEffect(() => {
     ;(async () => {
       setIsLoading(true)
-      const ethNetwork = await (state.wallet as ETHWallet)?.ethGetChainId?.()
+      setEthNetwork(undefined)
```

Comment:
I overlooked this on the initial work on that state field but what about setting this to null with the proper `string | null` typings on the state field?

```suggestion
      setEthNetwork(null)
```

Reply:
You're right, the true type of this should be:

```ts
const [ethNetwork, setEthNetwork] = useState<string | null | undefined>()
```

Except that the `undefined` is implicit when passing no args to `useState`. 

Fixed in https://github.com/shapeshift/web/pull/2626/commits/7d714e01f27a513dd6d0c77b8780755ad326d76e